doc/modules/linear_model.rst

```diff
@@ -735,7 +735,11 @@ of a single trial are modeled using a
 Logistic regression is implemented in :class:`LogisticRegression`. 
 This implementation can fit binary, One-vs-Rest, or multinomial logistic 
 regression with optional :math:`\ell_1`, :math:`\ell_2` or Elastic-Net 
-regularization. Note that regularization is applied by default.
+regularization.
+
+**Note that regularization is applied by default**, which is common in machine
+learning but not in statistics. No regularization amounts to setting C to a
+very high value (but with better numerical stability).
```

Comment:
I would still like to have the ``penalty='none'`` in there somehow. Also, right now I think it's unclear what has better numerical stability than what.

Reply:
so which one is true:

- using regularization is more stable than not using regularization
- in the unregularized case, using a high C is more stable than not using C at all

?